README.md

```diff
@@ -20,8 +20,10 @@ $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
 Definition of big O
 
 $f(n)\in O(g(n)) \iff \exists c>0, \exists n_0, \forall n\ge n_0: f(n) \leq c g(n)$
+
 For any c > 0 there is a $n_0$ so that $n \leq n_0$  so $f(n)<c⋅g(n)$
-Then for O(g(n)) a specific c and $n_0$ such that $f(n)\leq c⋅g(n)$
+Then for O(g(n)) there is a specific c and $n_0$ such that $f(n)\leq c⋅g(n)$
+
```

Comment:
Those are only the definitions, I don’t see any proof. Please proof the relation and you can use some concrete examples.